test/e2e/flowaggregator_test.go

```diff
@@ -212,6 +213,11 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	})
 	require.NoError(t, err, "IPFIX collector did not receive expected number of data records and timed out.")
 
+	if checkBandwidth {
+		// Wait one more export interval of flow aggregator to calculate a more accurate bandwidth value
+		time.Sleep(aggregatorExportInterval * time.Second)
+	}
+
```

Comment:
Is this required as you are already checking exportTime in L271? What is the purpose?

Reply:
These lines is used for getting the correct octetTotalCount.

Reply:
Could you address this comment? We do not want to add unnecessary sleep in e2e tests. It will increase the runtime of e2e tests, which is already high.